projects/cashmere/src/lib/pipes/pipes.module.ts

```diff
@@ -1,9 +1,10 @@
 import {NgModule} from '@angular/core';
 
 import {EllipsisPipe} from './ellipsis.pipe';
+import { FileSizePipe } from './file-size/file-size.pipe';
 
 @NgModule({
-    declarations: [EllipsisPipe],
-    exports: [EllipsisPipe]
+    declarations: [EllipsisPipe, FileSizePipe],
+    exports: [EllipsisPipe, FileSizePipe]
```

Comment:
Again, I think that all pipes need to have their own folder and module.  This will scale a lot better than having a single Pipes module.

Reply:
moved to its own module and imported it into the common cashmere module.